App.js

```diff
@@ -1,17 +1,16 @@
-import { StyleSheet, Text, View } from "react-native";
+import { NavigationContainer } from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import MainView from "./components/MainView";
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: "#fff",
-  },
-});
+const Stack = createNativeStackNavigator();
 
 const App = () => {
   return (
-    <View style={styles.container}>
-      <Text testID="header">Hello world</Text>
-    </View>
+    <NavigationContainer>
+      <Stack.Navigator>
+        <Stack.Screen name="home" component={MainView} />
```

Comment:
`<Stack.Navigator>
      <Stack.Screen
        name="Home"
        component={HomeScreen}
        options={{ title: 'My home' }}
      />
    </Stack.Navigator>`
   I would suggest a title with the name of the newspaper, Yesterdays News
   https://reactnavigation.org/docs/headers/

Reply:
`<Stack.Screen name="home" component={MainView} options={{ title: 'Yesterdays News' }} />`
like this